### PR TITLE
Add a health-check API

### PR DESCRIPTION
**Pull request by danielbeardsley**
Pull #5 changed it so an empty 'key' query param would 404 (previously
it would 200).

Let's open a specific url to just return a 200, no matter the query
param.

### CODE REVIEW

File: src/pixel-ping.coffee [hunk 1]
```diff
@@ -119,7 +119,10 @@ process.on 'uncaughtException', (err) ->
 # If it does, serve the pixel and record a hit.
 handleRequest = (req, res) ->
   params = url.parse req.url, true
-  if key = params.query?.key
+  if params.pathname == "/health-check"
+    res.writeHead 200, emptyHeaders
```
Comment (djmetzle) [inline src/pixel-ping.coffee]: Maybe we should use a 204?

Comment (danielbeardsley) [reply src/pixel-ping.coffee]: Derps, good point.

Comment (danielbeardsley) [reply src/pixel-ping.coffee]:
```suggestion
    res.writeHead 204, emptyHeaders
```

File: lib/pixel-ping.js [hunk 2]
```diff
@@ -176,7 +164,10 @@
   handleRequest = function(req, res) {
     var key, params, ref;
     params = url.parse(req.url, true);
-    if (key = (ref = params.query) != null ? ref.key : void 0) {
+    if (params.pathname === "/health-check") {
+      res.writeHead(200, emptyHeaders);
```
Comment (danielbeardsley) [inline lib/pixel-ping.js]:
```suggestion
      res.writeHead(204, emptyHeaders);
```

**Comment by danielbeardsley** [comment]
Fixed!

**Review by sctice-ifixit — APPROVED** [review]
CR :rose:

**Review by djmetzle — APPROVED** [review]
CR :gear:

**Comment by danielbeardsley** [comment]
QA :+1: 

image

**Pull request merged**